.github/workflows/manual-integration-test.yaml

```diff
@@ -1,6 +1,15 @@
 name: manual-run-integration-test
 on:
   workflow_dispatch:
+    inputs:
+      deployLatest:
+        description: 'Deploy unreleased latest?'
+        required: true
+        default: "false"
+        type: choice
```

Comment:
how about using the `boolean` type?

Reply:
updated.

I believe the change is correct, but https://github.com/actions/runner/issues/1483 is confusing.